student.rb

```diff
@@ -1,4 +1,6 @@
-class Student
+require_relative 'apprentice_teacher.rb'
+
+class Student < ApprenticeTeacher
```

Comment:
a student is not a type of ApprenticeTeacher. What's a better way to refactor these classes and use inheritance to DRY up the code?